virttest/virt_vm.py

```diff
@@ -1002,7 +1012,7 @@ def verify_kernel_crash(self):
                 return
             match = re.search(panic_re, data, re.DOTALL | re.MULTILINE | re.I)
             if match:
-                raise VMDeadKernelCrashError(match.group(0))
+                raise VMKernelCrashError(match.group(0))
```

Comment:
This change will cause some test case checkpoints to be failed, so with this change, we need to update test case part, not only tp-qemu, but also tp-libvirt.

Reply:
About tp-qemu and tp-libvirt cases failed.
I'm not sure if this step should be modified at our discretion.
This is more like a product issue causing automation cases to fail, 
and ultimately whether to keep this checkpoint may be up to the feature owner to decide.

Reply:
According to the offline discussion, I will send a depend patch 
to update the lower layer call about the exception.
thanks for reminding :)

Reply:
VMDeadKernelCrashError handles both Kernel trace and crash. Then why the change required?

Reply:
> VMDeadKernelCrashError handles both Kernel trace and crash. Then why the change required?

Please refer to the patch for the background: https://github.com/avocado-framework/avocado-vt/pull/3635

Reply:
Thanks a lot, I think your suggestion is very beneficial. 
After matching we can refine exception types. 
I'll commit the changes later.